lib/srv/desktop/tdp/proto_test.go

```diff
@@ -78,52 +74,6 @@ func TestBadDecode(t *testing.T) {
 	require.Error(t, err)
 }
 
-var encodedFrame []byte
-
-func BenchmarkEncodePNG(b *testing.B) {
```

Comment:
can we get a corresponding test in the rust code? Or, if not a benchmark (is that still on nightly?), then a test somewhere? I'm actually not familiar with tests in our rust code, but I see we have a few.

Reply:
I've added a criterion's benchmark to the Rust library